sawsanias/gh-explorer
# #4: implement ResultsPanel (closes #4)

Issue #4
## Test Plan
### tests performed

To test implementation of the three search results cases, a dummy json is created. Start the app and type the url http://localhost:8080/#/search ,
1. you should see a list of results
   try to change the browser window size to make it smaller so you can see the scroll bar inside the search result panel
2. try to assign "undefined" to the repositories variable in routes/Search.js , refresh the page you should see "Failed search"
3. change the repositories variable to empty array, refresh the page, you should see "No results" message in the search panel.
### tests not performed (domain coverage)

> At times not everything can be tested, and writing what hasn't been tested is just as important as writing what has been tested.
> 
> An example of partial test is a field displaying 4 possible values. If 3 values are tested, with screenshots, and 1 is not, then it should be mentioned here.}
